daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -68,13 +68,15 @@ tests tmpDir damlTypesDir = withSdkResource $ \_ -> testGroup "Integration tests
     , testCase "daml new --list" $ callCommandQuiet "daml new --list"
     , packagingTests
     , quickstartTests quickstartDir mvnDir
+    , createDamlAppTests createDamlAppDir
     , cleanTests cleanDir
     , templateTests
     , deployTest deployDir
     , fetchTest tmpDir
     , codegenTests codegenDir damlTypesDir
     ]
     where quickstartDir = tmpDir </> "q-u-i-c-k-s-t-a-r-t"
+          createDamlAppDir = tmpDir </> "createApp"
```

Comment:
Just create a new temp dir for the tests that need this. No need to copy more of the mess that we made here for new tests.